std-bits/base/src/main/java/org/enso/base/enso_cloud/audit/AuditLogMessage.java

```diff
@@ -8,7 +8,7 @@
 import org.enso.base.CurrentEnsoProject;
 import org.enso.base.enso_cloud.CloudAPI;
 
-public class AuditLogMessage implements AuditLogAPI.LogMessage {
+public class AuditLogMessage implements AuditLogApiAccess.LogMessage {
```

Comment:
A public class extends non-public class!? It is not really nice to expose such internals in an API.

Reply:
Hmm, you are right, `AuditLogMessage` does not need to and probably should never have been public, I will make it package-private.